core-bundle/src/Intl/Locales.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Intl;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\System;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class Locales
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    /**
+     * @var ContaoFramework
+     */
+    private $contaoFramework;
+
+    /**
+     * @var array<string>
+     */
+    private $localesList;
+
+    /**
+     * @var array<string>
+     */
+    private $backendLocales;
+
+    public function __construct(TranslatorInterface $translator, RequestStack $requestStack, ContaoFramework $contaoFramework, array $localesList, array $backendLocales)
+    {
+        $this->translator = $translator;
+        $this->requestStack = $requestStack;
+        $this->contaoFramework = $contaoFramework;
+        $this->localesList = $localesList;
+        $this->backendLocales = $backendLocales;
```

Comment:
should this be renamed back as well? Also, the variable is currently unused.

Reply:
> should this be renamed back as well?

No.

> Also, the variable is currently unused.

https://github.com/contao/contao/blob/8796a68c24a69e8f7c2e36c0ca205b684fe6d4ba/core-bundle/src/Intl/Locales.php#L209